samples/tutorial/Tutorial1.md

```diff
@@ -90,8 +90,8 @@ object WelcomeWorkflow : StatefulWorkflow<Unit, State, Output, WelcomeScreen>()
   object Output
 
   override fun render(
-    renderProps: Unit,
-    renderState: State,
+    props: Unit,
```

Comment:
I know the template creates them as `renderX`, but everywhere else in the code you reference them as just `x`.

Reply:
`renderX` is the new API. See #307 and #322 .